python_modules/libraries/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing Libraries
```

Comment:
We generally let the software that renders the page handle the word wrap in our MD files (e.g. in CHANGES.md and README.md).